### ISSUE TITLE
Is the project still maintained

### ISSUE THREAD

**Issue by numbnut** [pablojim/highcharts-ng#482]
Hi,
is this project still maintained and stable?
It would be nice to clarify that in the readme.
Thanks!

**Comment by kalouantonis** [comment]
I'd like to know this too. There are open pull requests that have been sitting around without activity since Feb 4, 2015.

**Comment by pablojim** [comment]
Thanks for the interest. I maintain this project when I get the time free from other commitments. I hope to have more time for maintaining it in the future but as with all side projects I can't commit to this.

On a general point I think that there is a slight conflict with this plugin between adding more and more support for features and keeping it simple and doing the most common tasks well. Though this could be spelt out better!

So some plans to improve things:

- Pull requests are always welcome - sometimes they don't come up to the standard to merge and require significant effort to fix up. Sometimes the submitters help with this but often they are "drive by pull requests" where the person has fixed their particular problem and stops responding.
- I'm more than happy to share the workload with others. Ideally a group of committers/reviewers. If some has a few pull requests that are sensible I'd be more than happy to add them as a committer to the project.
- I think a triage of issues/PRs would help too and a roadmap would help too.  @bflopez @kalouantonis @numbnut are you guys willing to help with this? 

So in short, any help gratefully received!

**Comment by numbnut** [comment]
Thanks for the reply! I'm sorry, but I can't contribute or help to maintain this project.
Good luck anyway!

**Comment by kalouantonis** [comment]
Thanks for clarifying. I can't commit to maintaining this project, but I will contribute pull requests for issues that crop up or features that align with my company's needs as we're using it on production.

**Comment by bflopez** [comment]
I would love to help! It will take me a little while getting to know the library a little better. We can talk more about a roadmap/plan. We might find out that there are other ways to make the library great for everyone.

**Comment by pablojim** [comment]
@bflopez++

**Comment by Rafi993** [comment]
It would be better if the roadmap/plan is drafted and updated in readme. It would be Ideal to get the chartconfig object same as highcharts so as to avoid all confusion and problems that are specific to highchars ng
